chart/kubeapps/templates/oauth2-secret.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.authProxy.enabled }}
```

Comment:
This flag means that the configuration will assume *an* auth proxy is enabled, but not necessarily Kubeapps' managed oauth2-proxy. The help string for `enabled` tries to be clear about the limit of this flag on its own:

```
  ## Set to true if Kubeapps should configure the OAuth login/logout URIs defined below.
```

So I think you want the same condition which is used in the deployment to include the oauth2-proxy:

```
        {{- if and .Values.authProxy.enabled (not .Values.authProxy.external) }}
```

Reply:
good point, I missed that.